CHI_TECH/ChiMath/Quadratures/LegendrePoly/legendrepoly.cc

```diff
@@ -1,10 +1,12 @@
 #include "legendrepoly.h"
 #include <math.h>
+
```

Comment:
Even though it's not really part of this diff, please change
#include<math.h>
to
#include<cmath>